src/@types/types.d.ts

```diff
@@ -140,9 +140,7 @@ export type RemoteModule = {
 export type ChromeModule = {
   manifestLocation: string;
   ssoUrl?: string;
-  config?: {
-    ssoUrl: string;
-  };
+  config?: Record<string, unknown>;
```

Comment:
As this is not a public type I think we might want to be as specific as possible. Just so we know what we might expect from apps to come. So far we only have ssoUrl and the profile requirement coming.

Reply:
Ah, good point!